### PR TITLE
PNI hero changes

### LINKED ISSUE #5411: PNI - update hero banner
- [x] move big heading and intro copy to hero
- [x] adjust hero banner height
- [x] adjust alignment

image

---
###  Design -> Dev Handoff Checklist

- [x] Design has been finalized
- [x] Link to related ticket: #5314
- [x] Link to Abstract (make sure it can be viewed by anyone)
- [x] Design changes been merged into the master Sketch file
  - Merged into master Sketch file via Abstract. (_Master > Buyers Guide.sketch > [Home Page Updates (Q4 2020](https://share.goabstract.com/07a9aca5-3903-4f00-828a-6ac8bebb5be3?sha=e49d77cc9da6a70e8c412cd985e28138a3091cd5))_).
- [x] Desktop and mobile mockups are included
  - [[Desktop] Homepage - Index](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=103e293c-16c0-48af-bac2-dc7d7545e5ea&mode=design)
  - [[Mobile] Homepage](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=3f690fff-c322-4f11-8505-04aee49fea04&mode=design)
- [x] Name of the Sketch file (and specific page/section to look at, if applies)
- [x] Assets are included. Link: 
  - background: https://drive.google.com/open?id=1AYYp44yBQ0wY8i57RP3F1fBOmcaqqDSL&authuser=sabrina%40mozillafoundation.org&usp=drive_fs
  - webby award: https://drive.google.com/open?id=14RyywjOt5pBnl_IRg9jCer7hmoCfoC-J&authuser=sabrina%40mozillafoundation.org&usp=drive_fs
- [x] Other states for elements are designed: The background would need to expand to accommodate the text on various screen widths and languages

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5411 

https://foundation-s-issue-5411-19axhq.herokuapp.com/en/privacynotincluded/

### CODE REVIEW

**Review by sabrinang — CHANGES_REQUESTED** [review]
- keep using body large for intro text (same as what was live 20px / 30px / 300)

- Switch to new background from issue which will provide better contrast for the text: https://drive.google.com/open?id=1AYYp44yBQ0wY8i57RP3F1fBOmcaqqDSL&authuser=sabrina%40mozillafoundation.org&usp=drive_fs

File: network-api/networkapi/buyersguide/templates/catalog.html [hunk 1]
```diff
@@ -9,43 +9,24 @@
...
   <!-- home banner as css background -->
-  <div class="container badge-container">
-    <div class="row h-100">
-      <div class="col-12 col-md-10 offset-md-1 col-xl-12 offset-xl-0 d-flex justify-content-end">
+  <div class="container">
+    <div class="row">
+      <div class="col-12 col-lg-8 offset-lg-2 intro-text order-2 order-lg-1">
+        <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
+        <p class="mb-0">
+          {% blocktrans trimmed %}
```
Comment (KalobTaulien) [inline network-api/networkapi/buyersguide/templates/catalog.html:17]: Nice use of `trimmed`!

**Review by KalobTaulien — APPROVED** [review]

**Comment by mmmavis** [comment]
@sabrinang thanks for the feedback. PR has been updated accordingly!

**Review by sabrinang — APPROVED** [review]
Great thanks for revising the type and image, looks good!

**Pull request merged**